ci/unit/test_nlt_node.sh

```diff
@@ -24,5 +24,18 @@ sudo bash -c ". ./utils/sl/setup_local.sh; ./utils/setup_daos_server_helper.sh"
 # ./utils/node_local_test.py --max-log-size ????MiB --dfuse-dir /localhome/jenkins/ \
 #			   --server-valgrind all
 
+# Use the latest version that CI has available.
+python3.9 -m venv venv
```

Comment:
test_main_node.sh references `python3` but this references `python3.9`. We should be consistent if possible

Reply:
See above.  This tests the pydaos install process, storage_estimator still needs work to migrate unit_test.